src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -81,20 +100,45 @@ export const GettingStarted = () => {
             <h1>{title}</h1>
           </EuiTitle>
           <EuiSpacer size="s" />
-          <EuiText color="subdued" size="s" textAlign="center">
+          <EuiText size="m" textAlign="center">
             <p>{subtitle}</p>
           </EuiText>
           <EuiSpacer size="s" />
           <EuiSpacer size="xxl" />
-          <EuiFlexGrid columns={3} gutterSize="xl">
+          <EuiFlexGrid columns={4} gutterSize="l">
+            <EuiFlexItem>
+              <GuideCard
+                useCase="search"
+                guides={guidesState}
+                activateGuide={activateGuide}
+                isDarkTheme={isDarkTheme}
+                addBasePath={http.basePath.prepend}
+              />
+            </EuiFlexItem>
             <EuiFlexItem>
-              <UseCaseCard useCase="search" />
+              <GuideCard
+                useCase="observability"
+                guides={guidesState}
+                activateGuide={activateGuide}
+                isDarkTheme={isDarkTheme}
+                addBasePath={http.basePath.prepend}
+              />
             </EuiFlexItem>
             <EuiFlexItem>
-              <UseCaseCard useCase="observability" />
+              <ObservabilityLinkCard
+                navigateToApp={application.navigateToApp}
+                isDarkTheme={isDarkTheme}
+                addBasePath={http.basePath.prepend}
+              />
             </EuiFlexItem>
             <EuiFlexItem>
-              <UseCaseCard useCase="security" />
+              <GuideCard
```

Comment:
Can we render these cards in a loop? It looks like (almost) all are the same

Reply:
We need to display a guide card for search, a guide card for observability, a link card for observability and a guide card for security, so I don't think a loop would work with this logic.

Reply:
Would something like this not work?
```
{ ['search', 'observability', 'observabilityLink', 'security'].map((useCase) => {
      if (useCase === 'observabilityLink') { 
         return (<EuiFlexItem>
                <ObservabilityLinkCard
                  ....
                />
            </EuiFlexItem>) ;
      } 
      return (<EuiFlexItem>
         <GuideCard useCase={useCase} .../>
       </EuiFlexItem>);
    };
}
```